docker-compose.yml

```
@@ -16,19 +15,36 @@ services:
         ports:
             - "9998:9998"
 
+    rabbitmq:
+        image: rabbitmq
+        ports:
+          - "5672:5672"
+        restart: always
+
+    dramatiq:
+        build:
+            context: .
+            dockerfile: dramatiq.Dockerfile
```

`dramatiq.Dockerfile` e `Dockerfile` geram imagens praticamente iguais. Talvez a gente não precisa de duas imagens e e de dois `Dockerfiles`s. Aqui seria só usar a mesma imagem e “trocar” o `CMD` com a chave `command` do YAML:

```suggestion
        build: .
        command: ["dramatiq", "datasets.tasks", "-p3", "-t2", "-v"]
```

---

Ótima dica, acho que ficou bom!